### PR TITLE
Upgrade eletron to 11.2.1

### PR DESCRIPTION
**Pull request by jameskerr**
The React DevTools are broken in version 8 - 11. The fix is in electron master. This helps us get up to speed when the fix is in.

### CODE REVIEW

**Review by mason-fish — APPROVED** [review]
I'd love to get those dev tools back! lgtm, assuming those failing iTests aren't related to the bump

**Comment by jameskerr** [comment]
Oh shoot, we need to upgrade Spectron as well.

**Pull request merged**
